.github/PULL_REQUEST_TEMPLATE/small-change.md

```diff
@@ -9,6 +9,12 @@
 
 Not applicable.
 
+## Effects on driving behavior
+
+<!-- Describe how this PR affects the outputs of driving behavior. -->
+
+No effects.
```

Comment:
I think it sometimes has a little effect, so rephrasing it to something might be better.

```suggestion
Not applicable.
```